### ISSUE TITLE
ctc_* api support custom blank_index

### ISSUE THREAD

**Issue by zh794390558** [tensorflow/tensorflow#30444]
since `ctc_loss_v2(labels, logits, label_length, logit_length, logits_time_major=True, unique=None, blank_index=None, name=None) ` has supporting `blank_index`,  I think the `ctc_greed_decode_v2` and `ctc_beam_search_decode_v2` also should support it.

**Comment by ravikyram** [comment]
@zh794390558 ,
Thank you for reaching out. Can you please confirm if you are referring to `tf.nn.ctc_greedy_decoder
` instead of `ctc_greed_decode_v2`, as we couldn't find this `ctc_greed_decode_v2`.

**Comment by zh794390558** [comment]
@ravikyram  Sorry, ` tf.nn.ctc_greedy_decoder` is correct.

**Comment by rmothukuru** [comment]
@zh794390558,
Sorry for the delayed response. Can you please specify the use cases in which adding `ctc_greedy_decoder` will be helpful? Thanks!
